pages/portfolio.html

```diff
@@ -4,9 +4,36 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
+  <title>Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
 <body>
+  <h1><b>Projects</b></h1>
+  <section>
+  <h2><i>Unit 1</i></h2>
+    <nav>
```

Comment:
`nav` is used for areas whose main focus is providing navigation. From MDN
>The `<nav>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Common examples of navigation sections are menus, tables of contents, and indexes.

The main purpose here is to provide information about your projects, not navigating the site.